### PR TITLE
download_message: Remove CMS references from the message of big downl…

### PR DESCRIPTION
**Pull request by psaiz**
…oads. Increase the limit to 1GB.

Addresses https://github.com/cernopendata/opendata.cern.ch/issues/3664

Screenshot 2024-08-30 at 16 42 06

### CODE REVIEW

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js [hunk 1]
```diff
@@ -51,10 +51,10 @@ export default function DownloadWarningModal({
           connection, it may take several hours to download it.
         </p>
         <p>
-          Moreover, if you use one of the provided{" "}
-          <a href="/VM/CMS">Virtual Machines</a> to perform your analyses, then
-          you don't need to download datasets manually, because the VM will
-          fetch all the necessary file chunks via the XRootD protocol.
+          Most experiments provide virtual machines or containers to perform analyses.
```
[the remark this answers is not in the document]
Comment (psaiz) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js:54]: ok, done

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js [hunk 2]
```diff
@@ -51,10 +51,10 @@ export default function DownloadWarningModal({
...
+          If you use one of those, then you do not need to download datasets manually,
+          because all the necessary file chunks will be accessed via the XRootD protocol.
```
[the remark this answers is not in the document]
Comment (psaiz) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js:56]: ok, done

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js [hunk 3]
```diff
@@ -51,10 +51,10 @@ export default function DownloadWarningModal({
...
+          Check the <a href="/search?q=&f=type%3ADocumentation%2Bsubtype%3AGuide">getting started</a> for more details.
```
[the remark this answers is not in the document]
Comment (psaiz) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js:57]: ok, done

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiborsimko) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js]: We have `<code>foo</code>` in many places in the docs to refer to files, e.g. https://opendata-qa.cern.ch/docs/delphi-guide-docker

[the remark this answers is not in the document]
Comment (tiborsimko) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js]: BTW what about the suggested expanded verbiage above? ("Most colaborations...")

File: cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js [hunk 4]
```diff
@@ -51,10 +51,10 @@ export default function DownloadWarningModal({
...
         </p>
         <p>
-          Moreover, if you use one of the provided{" "}
-          <a href="/VM/CMS">Virtual Machines</a> to perform your analyses, then
-          you don't need to download datasets manually, because the VM will
-          fetch all the necessary file chunks via the XRootD protocol.
+          Most collaborations provide container images or virtual machine images allowing to perform analyses.
+          If you use one of those, then you do not need to download datasets manually,
+          because all the necessary file chunks will be accessed via the XRootD protocol during the live analysis.
+          Please check the corresponding <a href="/search?q=&f=type%3ADocumentation%2Bsubtype%3AGuide">getting started</a> guides for more details.
```
Comment (tiborsimko) [inline cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js:57]: Note that the link I suggested had `q=getting+started` to limit the searhc only to getting started guides.

Comment (psaiz) [reply cernopendata/modules/theme/assets/semantic-ui/js/records/components/DownloadWarningModal.js]: I missed that one, sorry. I've put it now.

**Review by tiborsimko — APPROVED** [review]

**Pull request merged**